FETraining Assignment 6

dkdeepesh opened the pull request:
Assignment 6
Desktop: -
![image](https://github.com/user-attachments/assets/3aa77556-0439-481f-b92f-2d0a0067117f)
Mobile: -
![image](https://github.com/user-attachments/assets/6ef3bd70-0065-4f39-bc0a-519968a9901f)
Result: - 
Desktop: -
![image](https://github.com/user-attachments/assets/b3997a1b-00bf-4f06-8953-79a680ccebe4)
Mobile: -
![image](https://github.com/user-attachments/assets/2a7b72ff-e526-4011-b02e-f3520f615a4a)
![image](https://github.com/user-attachments/assets/21081cbf-1c78-4a9b-9fa7-6219532c0291)

mathrik commented:
<img width="430" alt="Screenshot 2024-07-12 at 3 26 32 PM" src="https://github.com/user-attachments/assets/ea09944f-78f6-467e-b67f-b1393d2ff375">

It looks like the assignment asked for this text to use a white font color and it would show up a lot better in a lighter color.

dkdeepesh commented on Assets/scss/components/_assignment.scss:
```diff
@@ -0,0 +1,178 @@
+        padding: 50px;
+
+        h1 {
+            font-size: 32px;
```
Sure, I will do the changes.

dkdeepesh replied:
@zmanning29  @mathrik I did the require changes.

Lalit-gour1 commented:
@dkdeepesh can you please add screenshot of the result of your code.
Cc:- @jvaishnavrp

mathrik commented:
<img width="1623" alt="Screenshot 2024-07-15 at 10 13 09 AM" src="https://github.com/user-attachments/assets/fde525f3-7d13-4e09-8ce9-94b485cc7a5d">
 This is what I see currently when I checkout your branch.

dkdeepesh commented:
@mathrik I have added screenshot of mobile and desktop after doing suggested changes from you and in that I have changed the color with white color, could you please try again by pulling latest changes.

mathrik commented:
I see the text is white now. Are you still working on the buttons?

mathrik commented:
@Lalit-gour1 have you talked to the team about using semitransparent backgrounds underneath text when it's overlaid on an image? That would help the readability a lot for this ticket.

Lalit-gour1 commented:
> @Lalit-gour1 have you talked to the team about using semitransparent backgrounds underneath text when it's overlaid on an image? That would help the readability a lot for this ticket.

@mathrik  I forgot to add the gradient value that'll be on image, somehow I missed it so from top it'll be background: linear-gradient(to top, rgba(49, 54, 55, 0), #313637);
From bottom:-  linear-gradient(to bottom, rgba(49, 54, 55, 0), #313637)
@dkdeepesh  please use this image as well to make it just like spec
![home_image](https://github.com/user-attachments/assets/cedef020-2d08-4b3b-9b00-f420ae2a9c7a)
Cc:- @jvaishnavrp

Lalit-gour1 commented:
@dkdeepesh  Both of your button has transparent bg color while in design one button has white bg and other button has transparent bg. Also on hover please do the opposite the transparent button should have bg color as white and white bg button should be transparent bg on hover.
Cc:- @jvaishnavrp

Lalit-gour1 commented on Assets/scss/components/_assignment.scss:
```diff
@@ -0,0 +1,189 @@
+    .image-area {
+        width: 67%;
+        position: relative;
+        background-image: url('../../images/home_image.png');
```
@dkdeepesh  Why you are using background-image instead of img tag, because later on we are implementing the same comp in AEM and it should be an authorable field. if you want to use background-image then store value in a css variable in inline css of that html element. 
Cc:-  @jvaishnavrp  @msharmaRP

dkdeepesh replied:
I have created a new PR for the above solution, incorporating all the required changes as requested.
https://github.com/jvaishnavrp/FETraining/pull/31
Cc: - @mathrik @Lalit-gour1 @zmanning29 @jvaishnavrp
